python/examples/face_detection_pytorch/detect_image.py

```diff
@@ -1,16 +1,17 @@
 import os
 import sys
-work_dir = os.path.abspath('./Ultra-Light-Fast-Generic-Face-Detector-1MB')
-sys.path.append(work_dir)
 from vision.ssd.config.fd_config import define_img_size
 from vision.ssd.mb_tiny_RFB_fd import create_Mb_Tiny_RFB_fd, create_Mb_Tiny_RFB_fd_predictor
 from utils import base64_to_ndarray
 
+
 threshold = 0.7
 candidate_size = 1500
 define_img_size(640)
 test_device = 'cpu'
 
+work_dir = os.path.abspath('./Ultra-Light-Fast-Generic-Face-Detector-1MB')
```

Comment:
The following error happens in my environment. Do you see what is wrong?
```
No module named 'vision'
Traceback (most recent call last):
  File "<string>", line 12, in <module>
ModuleNotFoundError: No module named 'vision'
```

Reply:
oh, you're right. that's why you add the path first. I was under the impression that was a dependency installed through `pip`.